JGActionSheet/JGActionSheet.m

```diff
@@ -600,21 +604,31 @@ - (void)tapped:(UITapGestureRecognizer *)gesture {
 }
 
 - (void)orientationChanged {
-    if (_targetView && !CGRectEqualToRect(self.bounds, _targetView.bounds)) {
-        disableCustomEasing = YES;
-        [UIView animateWithDuration:(iPad ? 0.4 : 0.3) delay:0.0 options:UIViewAnimationOptionBeginFromCurrentState | UIViewAnimationOptionCurveEaseInOut animations:^{
-            if (_anchoredAtPoint) {
-                [self moveToPoint:_anchorPoint arrowDirection:_anchoredArrowDirection animated:NO];
-            }
-            else {
-                [self layoutSheetInitial:NO];
-            }
-        } completion:^(BOOL finished) {
-            disableCustomEasing = NO;
-        }];
-    }
+    disableCustomEasing = YES;
+    [UIView animateWithDuration:(iPad ? 0.4 : 0.3) delay:0.0 options:UIViewAnimationOptionBeginFromCurrentState | UIViewAnimationOptionCurveEaseInOut animations:^{
+    } completion:^(BOOL finished) {
```

Comment:
It looks like you're using this for performSelector:afterDelay: is that right?  We're not trying to animate anything here anymore because we're just hiding/showing it.